script.js

```diff
@@ -1,56 +1,62 @@
-let bookList = [];
-
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
-
-function addBook(title, author) {
-  const book = new Book(title, author);
-  bookList.push(book);
-}
-
-function filterByTitle(item) {
-  return item.title !== 'title 1';
-}
-
-function addBookToStorage(title, author) {
-  window.localStorage.setItem(title, author);
-}
-
-function revomeFromStorage(title) {
-  window.localStorage.removeItem(title);
-}
-
-function removeBook(title, e) {
-  bookList = bookList.filter(filterByTitle);
-  e.parentNode.parentNode.removeChild(e.parentNode);
-  revomeFromStorage(title);
-}
-
-function display(title, author) {
-  const div = document.createElement('div');
-  div.innerHTML = `
-  <p>${title}</p>
-  <p>${author}</p>
-  <button onClick = "removeBook('${title}', this)">Delete</button>`;
-  const bookContainer = document.getElementById('bookContainer');
-  bookContainer.appendChild(div);
+class Books {
+  constructor() {
+    this.bookList = [];
+  }
+
+  addBook(title, author) {
+    const book = { title, author };
+    this.bookList.push(book);
+  }
+
+  static addBookToStorage(title, author) {
+    window.localStorage.setItem(title, author);
+  }
+
+  static display(title, author) {
+    const div = document.createElement('div');
+    div.innerHTML = `
+    <p>${title}</p>
+    <p>${author}</p>
```

Comment:
### hello @Team

- Good work on this part, but let's make it greater, add to them a `<h>` tag to make the visual of the author and the title clear for the reader.